src/05-derivatives/03-imaging.md

```diff
@@ -5,14 +5,119 @@ extent and resolution.
 
 ## Preprocessed, coregistered and/or resampled volumes
 
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        <datatype>/
+            <source_keywords>[_space-<space>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<ext>
+```
+
+
 Volumetric preprocessing does not modify the number of dimensions, and so
 the specifications in [Preprocessed or cleaned data][common_preproc]
 apply.
-In addition, all processed files include the following metadata JSON fields:
+The use of surface meshes and volumetric measures sampled to those meshes is
+sufficiently similar in practice to treat them equivalently.
```

Comment:
This could probably be worded better.

Reply:
Where do you see the problem is? (just to propose amends in the right direction)

Reply:
Just feels a little awkwardly worded. Maybe it could be written more persuasively. But I'm inclined to go for it and let people address wording later.